ambry-replication/src/main/java/com/github/ambry/replication/BackupCheckerThread.java

```diff
@@ -59,6 +72,22 @@ public class BackupCheckerThread extends ReplicaThread {
 
   private final Logger logger = LoggerFactory.getLogger(BackupCheckerThread.class);
   public static final String DR_Verifier_Keyword = "dr";
+  protected final AmbryCache fileDescriptorCache;
+  private final String basePath = "/export/content/lid/logs/ambry-server/i001";
```

Comment:
Please don't hardcode filepath. Use a config.

Reply:
Also can we use relative path instead of absolution path? Let's refer to how other modules/existing code picks the path.

Reply:
I checked the config on the host. They use absolute paths. Absolute paths eliminate any confusion where the files will land in the system.

Reply:
This is open source repo. There should be no linkedin specific assumption in this code.

Also, directory structure is more of a deployment concern. Tomorrow the infra could change, and the directory structure might change as well. So its never good to hardcode directory structures in code.

Reply:
I will remove the hardcoded path. I am not talking about that above.